server/api/stream.go

```diff
@@ -213,17 +214,23 @@ func LogStreamSSE(c *gin.Context) {
 	}
 
 	go func() {
-		err := server.Config.Services.Logs.Tail(ctx, step.ID, func(entries ...*model.LogEntry) {
-			for _, entry := range entries {
-				select {
-				case <-ctx.Done():
-					return
-				default:
-					ee, _ := json.Marshal(entry)
-					logChan <- ee
+		batches := make(logging.LogChan, 100)
```

Comment:
Is Server limited by 100 workflow execution simultaneously? Why 100? Should it be a constant?